jersey/connector/src/main/java/io/helidon/jersey/connector/HelidonProperties.java

```diff
@@ -27,8 +32,44 @@ private HelidonProperties() {
     }
 
     /**
-     * A Helidon {@link Config} instance used to create the corresponding {@link io.helidon.webclient.api.WebClient}.
+     * Property name to set a {@link Config} instance to by used by underlying {@link WebClient}.
      * This property is settable on {@link jakarta.ws.rs.core.Configurable#property(String, Object)}.
+     *      
+     * @see io.helidon.webclient.api.WebClientConfig.Builder#config(io.helidon.common.config.Config) 
      */
     public static final String CONFIG = "jersey.connector.helidon.config";
+
+    /**
+     * Property name to set a {@link Tls} instance to be used by underlying {@link WebClient}.
+     * This property is settable on {@link jakarta.ws.rs.core.Configurable#property(String, Object)}.
+     * 
+     * @see io.helidon.webclient.api.WebClientConfig.Builder#tls(Tls) 
+     */
+    public static final String TLS = "jersey.connector.helidon.tls";
+
+    /**
+     * Property name to set a {@code List<? extends  ProtocolConfig>} instance with a list of
+     * protocol configs to be used by underlying {@link WebClient}.
+     * This property is settable on {@link jakarta.ws.rs.core.Configurable#property(String, Object)}.
+     * 
+     * @see io.helidon.webclient.api.WebClientConfig.Builder#protocolConfigs(List) 
+     */
+    public static final String PROTOCOL_CONFIGS = "jersey.connector.helidon.protocolConfigs";
+
+    /**
+     * Property name to set a {@code Map<String, String>} instance with a list of
+     * default headers to be used by underlying {@link WebClient}.
+     * This property is settable on {@link jakarta.ws.rs.core.Configurable#property(String, Object)}.
+     *
+     * @see io.helidon.webclient.api.WebClientConfig.Builder#defaultHeadersMap(Map)
+     */
+    public static final String DEFAULT_HEADERS = "jersey.connector.helidon.defaultHeaders";
+
+    /**
+     * Property name to set a protocol ID for each request. You can use this property
```

Comment:
Maybe expand a bit on this property to mention that when TLS is used, we use ALPN, so it is not needed

Reply:
Done